AntWars/Board/BoardObjects.cs

```diff
@@ -59,79 +62,35 @@ public bool add(BoardObject boardObject)
             return true;
         }
 
-
-        private bool addToMap(BoardObject boardObject)
-        {
-            BoardObject[] objsInCoords = boardObjectList[boardObject.Coords.X,boardObject.Coords.Y];
-            if(objsInCoords == null)
-            {
-                objsInCoords = new BoardObject[0];
-            }
-            if(!containsType(objsInCoords, boardObject)) {
-                add(ref objsInCoords, boardObject);
-                boardObjectList[boardObject.Coords.X, boardObject.Coords.Y] = objsInCoords;
-                return true;
-            }
-            return false;
-        }
-
-
-        private bool containsType(IList<BoardObject> objs, BoardObject objectToCheck)
-        {
-            
-            foreach (BoardObject obj in objs)
-            {
-                if (obj == null) continue;
-                // the OR is a workaround due to Carry != Scout in type so we use our isAnt() method
-                if (obj.GetType() == objectToCheck.GetType() || (obj.isAnt() && objectToCheck.isAnt()))
-                {
-                    return true;
-                }
-            }
-            return false;
-        }
-
-        private void removeFromMap(BoardObject boardObject)
-        {
-            BoardObject[] objsInCoords = boardObjectList[boardObject.Coords.X, boardObject.Coords.Y];
-            remove(ref objsInCoords, boardObject);
-            boardObjectList[boardObject.Coords.X, boardObject.Coords.Y] = objsInCoords;
-        }
-
-
-        private void remove(ref BoardObject[] objs, BoardObject remove)
-        {
-            for (int i = 0; i < objs.Length; i++)
-            {
-                if (objs[i] == remove)
-                {
-                    objs[i] = null;
-                    if(i != objs.Length - 1)
-                    {
-                        BoardObject tmp = objs[objs.Length - 1];
-                        objs[i] = tmp;
-                    }
-                }
-            }
-            Array.Resize<BoardObject>(ref objs, objs.Length - 1);
-
-        }
-
```

Comment:
das ist weiter nach unten in der classes gegangen. Allgemein macht man die reihenfolge der methode/attribute je nach sichtbarkat 
public --> internal --> protected --> private